Makefile

```diff
@@ -476,7 +478,10 @@ endif
 
 test-netstandard: build-netstandard
 	@echo 🧪 TEST NETSTANDARD $(PROCESSOR) $(TARGET)
+	make test-netstandard-copy-output
+	dotnet test -a $(PROCESSOR) --configuration $(TARGET) ./bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup.Tests/ElectionGuard.ElectionSetup.Tests.csproj
```

Comment:
i pulled the election setup tests into the primary test workflow along with the decryption tests